slides/k8s/daemonset.md

```diff
@@ -73,18 +73,13 @@
 
 - Dump the `rng` resource in YAML:
   ```bash
-  kubectl get deploy/rng -o yaml --export >rng.yml 
+  kubectl get deploy/rng -o yaml >rng.yml
   ```
 
 - Edit `rng.yml`
 
 ]
 
-Note: `--export` will remove "cluster-specific" information, i.e.:
-- namespace (so that the resource is not tied to a specific namespace)
-- status and creation timestamp (useless when creating a new resource)
-- resourceVersion and uid (these would cause... *interesting* problems)
-
```

Comment:
Hmm, isn't this still the case?

Reply:
`--export` is going to be deprecated (https://github.com/kubernetes/kubernetes/pull/73787) so I thought it would be simpler to just ... not mention it at all. Again, in the interest of simplicity :)

Reply:
ah, makes sense.